localstack/testing/pytest/util.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pwd
+from multiprocessing import Process, ProcessError
+from typing import Callable
+
+
+def run_as_os_user(target: Callable, uid: str | int, gid: str | int = None):
+    """
+    Run the given callable under a different OS user ID and (optionally) group ID, in a forked subprocess
+    """
```

Comment:
Nit: would be nice if the method docstring would contain more information about the formats of uid/gid which are accepted (string as name and id as int, as far as i can see)